openwisp_monitoring/device/api/views.py

```diff
@@ -248,5 +265,31 @@ def _create_clients_graph(self, metric):
         graph.full_clean()
         graph.save()
 
+    def _create_resources_graph(self, metric, resource):
+        graph = Graph(metric=metric, configuration=resource)
```

Comment:
Let's add `if resource not in monitoring_settings.AUTO_GRAPHS` similarly to `_create_clients_graph()`